torch/quantization/fx/quantization_patterns.py

```diff
@@ -180,34 +189,52 @@ def convert(self,
 
 # tuple (activation_dtype, weight_dtype, compute_dtype)
 # these are supported types for common binary ops like add/mul etc.
-binary_op_all_dtypes = [
+all_dtypes = [
     (torch.quint8, torch.qint8, None),
     (torch.float16, torch.float16, None),
 ]
-binary_op_float16_dtypes = [
+fp16_dtypes = [
     (torch.float16, torch.float16, None)
 ]
-binary_op_int8_dtypes = [
+int8_dtypes = [
     (torch.quint8, torch.qint8, None),
 ]
 binary_op_supported_dtypes : Dict[Union[Callable, str], List[Tuple[torch.dtype, torch.dtype, None]]] = {
-    operator.add: binary_op_all_dtypes,
-    torch.add: binary_op_all_dtypes,
-    operator.mul: binary_op_all_dtypes,
-    torch.mul: binary_op_all_dtypes,
-    torch.bmm: binary_op_float16_dtypes,
-    torch.sub: binary_op_float16_dtypes,
-    operator.sub: binary_op_float16_dtypes,
-    torch.div: binary_op_float16_dtypes,
-    operator.truediv: binary_op_float16_dtypes,
-    torch.sum: binary_op_float16_dtypes
+    operator.add: all_dtypes,
+    torch.add: all_dtypes,
+    operator.mul: all_dtypes,
+    torch.mul: all_dtypes,
+    torch.bmm: fp16_dtypes,
+    torch.sub: fp16_dtypes,
+    operator.sub: fp16_dtypes,
+    torch.div: fp16_dtypes,
+    operator.truediv: fp16_dtypes,
 }
```

Comment:
should this variable now be named something like `fbgemm_and_qnnpack_binary_op_supported_dtypes`?

Reply:
this one actually contains ops that's not supported by fbgemm/qnnpack as well (fp16 ops) I guess this does not make much sense, we should probably be removing fp16_dtypes mapping here..

Reply:
added a comment for these, we'll remove them from this file some time in the future